terraform/staging/variable.tf

```diff
@@ -0,0 +1,9 @@
+variable "environment_name" {
+    type = string
+    default = "staging"
+}
+
+variable "project_name" {
+    type = string
+    default = "Housing-Staging"
```

Comment:
Do we want to hardcode our the project name in the template, if other projects are going to use this template then they might forget to change the project name (if it's different)